src/format/mod.rs

```diff
@@ -38,9 +38,19 @@ use offset::{FixedOffset, Offset};
 use {Datelike, Timelike};
 use {ParseWeekdayError, Weekday};
 
+#[cfg(all(feature = "locales", any(feature = "alloc", feature = "std", test)))]
+pub(crate) mod locales;
```

Comment:
huh, I didn't realize that we could use new syntax if it was cfg'd out and not in a different module, nice. (`pub(crate)` was [stabilized in 1.18.0](https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-1180-2017-06-08))

Reply:
Yes! :sweat_smile: working around the 1.18 was really tricky sometimes. The CI works great fortunately